### PR TITLE
upgrade to reCAPTCHA v3

### PR DESCRIPTION
**Pull request by xithan**

### CODE REVIEW

File: card/card.gemspec [hunk 1]
```diff
@@ -62,7 +62,7 @@ Gem::Specification.new do |s|
 
     # other
     ["diff-lcs",                   "~> 1.3"], # content diffs in histories
-    ["recaptcha",                  "~> 4.3"],
+    ["recaptcha",                  "~> 4.13.1"],
```
Comment (ethn) [inline card/card.gemspec]: issues with the more recent versions?

Comment (ethn) [reply card/card.gemspec]: anything worth a code comment?

Comment (xithan) [reply card/card.gemspec]: more recent? it's the most recent version

Comment (ethn) [reply card/card.gemspec]: DOH.

**Pull request merged**